home/pi/auto_run.sh

```diff
@@ -693,12 +697,7 @@ function setup_wizard() {
 
 function speak() {
     # Generate TTS audio using Mimic 1
-    ~/mycroft-core/mimic/bin/mimic -t $@ -o /tmp/speak.wav
-
-    # Play the audio using the configured WAV output mechanism
-    wavcmd=$( jq -r ".play_wav_cmdline" /etc/mycroft/mycroft.conf )
-    wavcmd="${wavcmd/\%1/\/tmp\/speak.wav}"
-    $( $wavcmd >/dev/null 2>&1 )
+    ~/mycroft-core/mimic/bin/mimic -t $@
```

Comment:
It's good that the user may hear some output as a result of this being called, but using mimic esp when it is not compiled for pulseaudio doesn't tell us much about whether e.g. mycroft-speak or the codepath that mycroft actually uses to do TTS will work for them, will it?